build_tools/cmake/iree_copts.cmake

```
@@ -120,6 +120,7 @@ iree_select_compiler_opts(IREE_DEFAULT_COPTS
     "-fvisibility=hidden"
   MSVC_OR_CLANG_CL
     "/DWIN32_LEAN_AND_MEAN"
+    "/DUSE_MATH_DEFINES"
```

Missing leading underscore?

```suggestion
    "/D_USE_MATH_DEFINES"
```

---

lol yeah, that'll do it. Thanks.